### PR TITLE
yamllint formatting cleanup for ocp-workload-pam7-offer-management-dmn-pmml

### PR DESCRIPTION
**Pull request by ritesh97-rh**
##### SUMMARY
For the specified roles:
- Create a yamllint file to enforce formatting rules
- Make changes to existing code base to adhere to formatting requirements
- yamllint errors will be displayed during formatting checks in future development

##### ISSUE TYPE
- Code Cleanup Request

##### COMPONENT NAME
ocp-workload-pam7-offer-management-dmn-pmml

### CODE REVIEW

File: ansible/roles/ocp-workload-pam7-offer-management-dmn-pmml/tasks/rhpam_openshift_dmn_pmml_workload.yml [hunk 1]
```diff
@@ -1,60 +1,104 @@
...
+    oc new-build --strategy docker
+    --image-stream="openshift/rhpam-kieserver-rhel8:7.5.0"
+    --binary --name rhpam-kieserver-rhel8-dmn-pmml
+    -n {{ OCP_PROJECT }}
 
 - name: "Configure Business Central ImageStream"
-  shell: "oc patch dc/{{ pam_app_name }}-rhpamcentr --type='json' -p '[{\"op\": \"replace\", \"path\": \"/spec/triggers/0/imageChangeParams/from/name\", \"value\": \"rhpam-businesscentral-rhel8-dmn-pmml:latest\"}]' -n {{ OCP_PROJECT }}"
+  shell: >-
+    oc patch dc/{{ pam_app_name }}-rhpamcentr --type='json'
+    -p '[{"op": "replace", "path": "/spec/triggers/0/imageChangeParams/from/name",
```
Comment (stencell) [inline ansible/roles/ocp-workload-pam7-offer-management-dmn-pmml/tasks/rhpam_openshift_dmn_pmml_workload.yml:27]: Have you tested removing the escaping of the `"` chars ?

**Review by stencell — CHANGES_REQUESTED** [review]
Overall this looks okay. I would like to know that it has been tested and deployed against an OpenShift cluster before merging. Has it been tested with the changes?

File: ansible/roles/ocp-workload-pam7-offer-management-dmn-pmml/tasks/rhpam_openshift_dmn_pmml_workload.yml (hunk 1 above)
Comment (ritesh97-rh) [reply ansible/roles/ocp-workload-pam7-offer-management-dmn-pmml/tasks/rhpam_openshift_dmn_pmml_workload.yml:27]: Checked via yaml to json conversion. Has not been conventionally tested. Would talk to Jonathan if you feel that is necessary - my time on this project ends on Friday.

Comment (stencell) [reply ansible/roles/ocp-workload-pam7-offer-management-dmn-pmml/tasks/rhpam_openshift_dmn_pmml_workload.yml:27]: @jkupferer ^

Comment (jkupferer) [reply ansible/roles/ocp-workload-pam7-offer-management-dmn-pmml/tasks/rhpam_openshift_dmn_pmml_workload.yml:27]: confirmed, looks good

File: ansible/roles/ocp-workload-pam7-offer-management-dmn-pmml/tasks/wait_for_deploy.yml [hunk 2]
```diff
@@ -15,16 +17,24 @@
   changed_when: false
   retries: "{{ deploy_status_retries }}"
   delay: "{{ deploy_status_delay }}"
-  until: 'rc_state.stdout |from_json | json_query("items[? (kind == ''ReplicationController'')].metadata.annotations.\"openshift.io/deployment-config.name\"") |intersect(item) |length >= 1'
+  until: >-
+    rc_state.stdout | from_json | json_query("items[?(kind
+    == ''ReplicationController'')].metadata.annotations."openshift.io/deployment-config.name"")
```
Comment (jkupferer) [inline ansible/roles/ocp-workload-pam7-offer-management-dmn-pmml/tasks/wait_for_deploy.yml:22]: This line break here in the middle of a `json_query` is hard to follow

**Review by jkupferer — CHANGES_REQUESTED** [review]
